### DISCUSSION TITLE
Controls values does not seems to dynamically update props in a component

### DISCUSSION THREAD

**Discussion by cichyadam** [storybookjs/storybook#16317] [Help]
Hi everyone!
I'm looking for a help with making the controls work in the storybook we use in our workplace.
We have updated storybook from `5.3.18` to `6.3.8` and I'm currently trying to get rid of knobs and use controls ase knobs are deprecated. I have followed the documentation for controls but I can't make it work. I have been struggling with this issue with our Components and couldn't find a way how to make it work so I decided to use the demo Button from storybook to try if that works and the behaviour is the same. This is how the story is defined based on article: https://medium.com/storybookjs/storybook-controls-ce82af93e430.

This is how I defined the story: 
```

import { Button } from '@storybook/react/demo'
import React from 'react'

export default {
  title: 'Components/Button',
  component: Button
}

export const Basic = (args) => <Button {...args} />

Basic.args = { children: "I'm a button" }
 
```
Here in a short video you can see that children control is not changing the props and components childrens remain the same as I defined in story.

https://user-images.githubusercontent.com/26605509/136925366-962e17e7-a3a4-445b-9601-03fe82380a00.mov

Thank you in advance for your help!

**Comment by shilman** [comment]
I tried your  exact code in a fresh project and it worked fine for me. Do you a have a reproduction repo you can share? If not, can you create one? See [how to create a repro](https://storybook.js.org/docs/react/contribute/how-to-reproduce).

**Reply by cichyadam** [reply]
Hi Michael, thanks for looking into it. I finally made the controls work for our storybook. The issue was that in `preview.js` I was using the method `addParameters` like this: 

```
import { addParameters } from '@storybook/react' 

addParameters({
  ...someParameters
})
```

Once I have rewritten `preview.js` to (see code below) it finally worked.   

```
export const parameters = {
  ...someParameters
} 
```
